library/panic_unwind/src/emcc.rs

```diff
@@ -90,7 +100,14 @@ pub unsafe fn panic(data: Box<dyn Any + Send>) -> u32 {
     if exception.is_null() {
         return uw::_URC_FATAL_PHASE1_ERROR as u32;
     }
-    ptr::write(exception, Exception { caught: AtomicBool::new(false), data: Some(data) });
+    ptr::write(
```

Comment:
The structure allocated above is too small, so this causes UB. This probably used to work because the allocator added some padding, but 4 bytes are now definitely to little for a structure of 12 bytes. `sz` should be `mem::size_of::<Exception>()`.

Reply:
Fixed